client/src/components/ScoreForm.js

```diff
@@ -95,34 +163,46 @@ const NewIntegerExercise = ({ value, units }) => {
 			step = 1		
 	} 
 	return (
-		<Form.Input type='number' width={8} min={0} max={9999} step={step} defaultValue={value} />	
+		<Form.Input
+			{...field.input} 
+			type='number' 
+			width={8} 
+			min={0} 
+			max={9999} 
+			step={step} 
+		/>	
 	)
 }
 
-const CurrTimerExercise = ({ value }) => {
-	const currTime = convertMinsToHrsMins(value)
-	return (
-		<div>{currTime}</div>
-	)
-}
+const NewTimerExercise = field => (
+	<Form.Input 
+	 	{...field.input}
+		type='number' 
+		label={field.label} 
+		labelPosition='right' 
+		width={4} 
+		min={field.min} 
+		max={field.max}
+	/>	
+)
 
-const NewTimerExercise = ({ value }) => {
-	let h = <Form.Input type='number' label='Hrs' labelPosition='right' width={4} min={0} max={99} />
-	let m = <Form.Input type='number' label='Mins' labelPosition='right' width={4} min={0} max={59} />
-	
-	return (
-		<Form.Group>
-		 {h}{m}
-		</Form.Group>
-	)
-}
+const mapStateToProps = (state) => {
+	return { 
+		initialValues: {
+			user: {
+				exercises:state.scoreForm.user.exercises, 
+				bonuses: state.scoreForm.user.bonuses
+			}
+		}
+	}
+}	
 
-const convertMinsToHrsMins = minutes => {
-	let h = Math.floor(minutes / 60)
-	let m = minutes % 60	
-	return (
-		h + '  Hrs  ' + m + '  Min'
-	)
+const mapDispatchToProps = (dispatch) => {
+  const { score } = ScoreFormActions;
+  return bindActionCreators({ score }, dispatch);
```

Comment:
Is this a stopgap? Do we want this in the final app? This is used when you want a child component to be independent of redux. Why are you using this?

Reply:
I don't know what you mean by stopgap. Simply copied from login.js, but it's not currently being used at the moment. This will need to be updated later once we have created the actions.

Reply:
I saw this: http://redux.js.org/docs/api/bindActionCreators.html